helm/chart/maesh/values.yaml

```diff
@@ -41,7 +41,10 @@ mesh:
   # Added so we can launch on nodes with restrictions
   nodeSelector: {}
   tolerations: []
-
+  forwardingTimeouts:
```

Comment:
I think that by default we should use the default values from Træfik, wdyt?

These options should be optional and commented as described [here](https://github.com/containous/maesh/blob/master/helm/chart/maesh/Guidelines.md). Something like:

```
# forwardingTimeouts:
#    dialTimeout: "30s"
#    responseHeaderTimeout: "0s"
#    idleConnTimeout: "1s"
```

Reply:
Good Catch!